Quicksort_3_way/Quicksort_3_way.c

```diff
@@ -0,0 +1,103 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+typedef void qsort3way_swap(void *a, void *b);
+typedef int qsort3way_cmp(void const *a, void const *b);
+
+static void qsort3way_aux(char *array_begin, char *array_end, size_t size,
+                          qsort3way_cmp *cmp, qsort3way_swap *swap) 
+{
+  if (array_begin < array_end)
```

Comment:
@Moddy2024  you kept only two spaces here , this should be four spaces. likewise change the entire code asap

Reply:
i changed the spaces @asha15  please review it